arcgisbinding/installing-arcgisbinding.qmd

```diff
@@ -25,21 +28,23 @@ If you are working in ArcGIS Pro 2.0 or beyond you have access to the built-in i
 - Select `Options` on the blue, left-hand side panel and in the pop-up window, under Application, select 'Geoprocessing'.
 - Under the R-ArcGIS Support options, select your desired R home directory.
 
-<!-- TODO Nick wants screen shot here --> 
+![](images/arcgisbinding-installer-homedir.png){width=50% fig-align="center" fig-alt="A screenshot of the Options window with the Detected R home directories drop-down in the R-ArcGIS Support section highlighted."}
 
 :::{.callout-warning collapse="false"}
 
 Note: All versions of R installed on your computer will appear in the drop-down menu. Make sure the version you select is R 3.2.2 or later. However, if you have installed R to a location other than the default, you might need to navigate to that location using the browse button.
 
 :::
 
-- If you have never installed the **arcgisbinding**, you will see a warning indicating that you need to install the package to enable R to connect with ArcGIS Pro. When you click on the icon next to the warning you will be presented with options to automatically download and install **arcgisbinding**, to separately download the package, or to install the package from file. Select the first option to `"automatically download and install"` the **arcgisbinding** package.
+- If you have never installed the **arcgisbinding**, you will see a warning indicating that you need to install the package to enable R to connect with ArcGIS Pro. When you click on the icon next to the warning you will be presented with options to install **arcgisbinding** from the internet, to download the latest versions of the package, or to install the package from file. Select the first option to `"Install package from the Internet"` the **arcgisbinding** package.
```

Comment:
we should remove "the" from in front of arcgisbinding

Reply:
> to download the latest versions of the package
Should be **version**

Reply:
done!